optimum/intel/openvino/quantization.py

```diff
@@ -526,7 +539,7 @@ def get_calibration_dataset(
 
     def _get_calibration_dataloader(
         self,
-        calibration_dataset: Dataset,
+        calibration_dataset: "Dataset",
```

Comment:
I think we can also soon deprecate the `input_names` attribute as it looks unused (to remove the constraint that `calibration_dataset` should have `column_names` attribute, making it optional)